Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -14,7 +14,7 @@
 using Stratis.Bitcoin.Utilities.FileStorage;
 using Transaction = NBitcoin.Transaction;
 
-[assembly: InternalsVisibleTo("Stratis.Bitcoin.Tests")]
+[assembly: InternalsVisibleTo("Stratis.Bitcoin.Features.Wallet.Tests")]
```

Comment:
Ah interesting I didn't think of doing it this way.
Is it not better to put it in an AssemblyInfo?